src/Command/MarkMigrated.php

```diff
@@ -60,7 +60,9 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption('skip-to', 'u', InputArgument::OPTIONAL, 'If present it will mark all versions until the given one')
+            ->addOption('up-to', 't', InputArgument::OPTIONAL, 'If present it will mark all versions including the given one');
```

Comment:
Regarding the `InputOption::VALUE_NONE`. All the others options are being setup using `InputArgument::OPTIONAL`. Is it right once they aren't `InputArgument`s?